pkgs/top-level/all-packages.nix

```diff
@@ -511,6 +511,8 @@ with pkgs;
     python3Packages = python34Packages;
   };
 
+  autoflake = callPackage ../development/python-modules/autoflake { };
```

Comment:
maybe:

```
pkgs/development/tools/analysis/
```

as a directory?

Reply:
Yep, I'll do that, thanks!

Or, maybe can I dig `pkgs/development/tools/python/` and use the directory?
Langs like `guile`, `haskell`, `ocaml`, `java`, .. seem to be doing that.
`go` is a notable exception and spreads `goimports`, `gotags` etc right under `developement/tools`.